resources/scripts/generate-email-template-if-no-recent-changes.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#
+# Script that checkout the given repository and
+# search for recent changes in the last x days for the given
+# file. If no then it will create a template email to be sent
+#
+# Requirements:
+#  - git, in order to clone the repository
+#  - gh, in order to send the existing list of open PRs.
+#
+
+set -euo pipefail
+
+REPO_URL=${1:?'Missing the GitHub repo URL'}
+BRANCH=${2:?'Missing the branch'}
+DAYS=${3:?'Missing the days since the file has not changed result'}
+
+EMAIL_FILE=email.txt
+if [ -e ${EMAIL_FILE} ] ; then
+  rm ${EMAIL_FILE}
+fi
+
+git clone "$REPO_URL" --branch "$BRANCH" "$BRANCH"
+
+cd "$BRANCH"
+
+if git --no-pager \
+    log --pretty=format: \
+      --name-only \
+      --since="${DAYS} days ago" \
+    | grep 'testing/environments/snapshot-oss.yml' ; then
+
+  echo 'nothing to be reported'
+else
+cat <<EOT > ../${EMAIL_FILE}
+Just wanted to share with you that the Elastic Stack version for the ${BRANCH} branch has not been updated for a while ( > ${DAYS} days).
+
+Those bumps are automatically merged if it passes the CI checks. Otherwise, it might be related to some problems with the
+Integrations Tests.
+
```

Comment:
Nit: Possibly mention where this script runs from so that people who read the email know what process generated it?

Reply:
Good point. The email should be from the automation user, similar to the one we sent with the flaky tests:

![image](https://user-images.githubusercontent.com/2871786/143430320-251c2803-8768-46b0-93d5-5e86232a9d58.png)


But probably it's worth to add a specific section to avoid any misleading. What kind of mentioning where you thinking?

>> Generated by the [job](url)

Reply:
> Generated by the job


Yeah, something like that. Basically, if I'm a consumer of the _result_ of any automation (like an email) I should be able to immediately know where to start looking to understand what process generated it. Pointing it just to the Jenkins job seems good. Something like:

This email was automatically generated by [Jenkins](https://github.com/elastic/apm-pipeline-library/pull/url)